pilot/pkg/proxy/envoy/discovery.go

```diff
@@ -133,6 +138,8 @@ var (
 	// while debouncing. Defaults to 10 seconds. If events keep
 	// showing up with no break for this time, we'll trigger a push.
 	DebounceMax time.Duration
+
+	debugDebounce = os.Getenv("PILOT_DEBOUNCE_DEBUG") != ""
```

Comment:
ugh!! really, more env vars?
please please move these to feature flags file.

Reply:
To be honest - I don't like the new interface, and doesn't fit well with how env variables are used. I don't see Go or Java having a central struct with all the env variables they support. Using env is supposed to be lightweight and local. It's still better than flags - and in master I don't mind moving it ( I have a PR in progress - and I just removed all the env variables since they're no longer needed)